spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialConvolution.scala

```diff
@@ -160,11 +164,14 @@ class SpatialConvolution[T: ClassTag](
       }
       var g = 0
       while (g < nGroup) {
+        val biasUse = if (withBias) {
+          bias.narrow(1, g * nOutputPlane / nGroup + 1, nOutputPlane / nGroup)
+        } else bias
```

Comment:
The logic just looks a little weird. what about

```scala
val biasUse = if (withBias) {
   ...
} else {
   null
}
```
So people will know it's a null instead of guessing what the bias is.